tardis/io/model_reader.py

```diff
@@ -7,6 +7,9 @@
 from astropy import units as u
 from radioactivedecay import Nuclide
 from radioactivedecay.utils import Z_DICT, elem_to_Z
+import h5py
```

Comment:
Import statements could be reordered to match [PEP 8 ordering](https://peps.python.org/pep-0008/#imports), but this won't affect functionallity